mlflow/models/evaluation/base.py

```diff
@@ -39,13 +39,23 @@
 class _ModelType:
     REGRESSOR = "regressor"
     CLASSIFIER = "classifier"
+    QUESTION_ANSWERING = "question-answering"
+    TEXT_SUMMARIZATION = "text-summarization"
+    TEXT = "text"
+    # TODO: Add 'retrieval' model type
```

Comment:
Do we have an internal ticket to track this? Retrieval would be an excellent next thing to add

Reply:
Created a ticket!